app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h3>Users Details</h3>
+
+<div class="user_find_container">
+  <div class="user_img_div">
+    <img class="user_img" src='<%= @user.photo_link %>'>
+  </div>
+    <div class="user_internal">
+        <h1><%= @user.name %></h1>
+        <p>Number of Posts: <%= @user.posts_counter %></p>
```

Comment:
- Good job done so far, but you do not have an attribute called `posts_counter` in the user model, please fix this 🙏